### PR TITLE
Set dimensions on setup and resize

### PR DESCRIPTION
**Pull request by jw-kaurand**
Sets dimensions when we resize via the api and during startup. This fixes an issue where the viewable area analytics parameter was not being set correctly on the startup ping.

JW7-2740

### CODE REVIEW

File: src/js/view/view.js [hunk 1]
```diff
@@ -802,7 +802,7 @@ define([
         this.resize = function(width, height) {
             var resetAspectMode = true;
             _resize(width, height, resetAspectMode);
-            _responsiveListener();
+            _setContainerDimensions();
```
Comment (pajong) [inline src/js/view/view.js]: ```_responsiveListener``` cancels the previous animationFrame, so it is required to be in resize.

Comment (jw-kaurand) [inline src/js/view/view.js]: Hmm, yes, though we want to execute _setContainerDimension immediately so we can update container sizes right away.  I think I can cancel any pending RAF calls in _setContainerDimensions for safety.

File: src/js/view/view.js [hunk 2]
```diff
@@ -53,6 +53,8 @@ define([
             _rightClickMenu,
             _resizeMediaTimeout = -1,
             _resizeContainerRequestId = -1,
+            _resizeRAFFunc = window.requestAnimationFrame || function(rafFunc) { return window.setTimeout(rafFunc, 17); },
```
Comment (pajong) [inline src/js/view/view.js]: The name of the function is really hard to follow what that function is doing - can we rename this to something more intuitive or leave a comment on what resizeRAFFunc means? Same for ```_resizecCancelRAFFunc```.
Also, don't forget to rebase your commits :)

Comment (pajong) [inline src/js/view/view.js]: Also note that the build is failing because the line is too long.

**Comment by jw-kaurand** [comment]
Closed in favor of https://github.com/jwplayer/jwplayer/pull/1323
